src/randgraphs.jl

```diff
@@ -38,6 +42,7 @@ or the  `Graph(nv, ne)` constructor, which randomly select `ne` edges among all
 edges.
 """
 function erdos_renyi(n::Integer, p::Real; is_directed=false, seed::Integer=-1)
+    @assert 0 <= p <= 1
```

Comment:
1) This should be a more informative error to the user, I think.
2) We should not be changing functionality in a PR that is supposed to be about docstring enhancements.